neuralpp/experiments/pimt_experiment.py

```diff
@@ -19,15 +19,15 @@
 mu2 = BasicVariable("mu2", float)
 
 # P(x, mu1, mu2) = Normal(x | mu1, 1.0) * Normal(mu1 | mu2, 1.0) * Normal(mu2 | 0.0, 1.0) propto
-formula = get_normal_piecewise_polynomial_approximation(x, mu1, 1.0)
-joint_simple = get_normal_piecewise_polynomial_approximation(x, mu1, 1.0) \
+formula = get_normal_piecewise_polynomial_approximation(x, mu1, 1.0, generator=mu1)
```

Comment:
I added a reference to SymPy's document. I believe "generator" is a good name since it's almost variable/indeterminate but slightly different (plus, "variable" is a more ambiguous name here). I agree with your point that it's not very easy to understand. But we do need this parameter for Poly representation to avoid later conversion.